### DISCUSSION TITLE
How setup next to have defaultLocale only available without prefix?

### DISCUSSION THREAD

**Discussion by R4F44L** [vercel/next.js#65882] [Help]
### Summary

I have a clean nextjs pages router app with en-GB and fr-FR locales. En-GB is default. 
I have a page on `test-path`. 
Right now I can access it by:
`/en-GB/test-path`
`/fr-FR/test-path`
`/test-path`
How can I exclude paths containing en-GB - make default locale available only without the locale prefix?

### Additional information

```js
next version 13.1.6
```

**Comment by TouhidZaman** [comment]
Try adding the following code blocks in your next.config.js file 

```
module.exports = {
  i18n: {
    locales: ['en-GB', 'fr-FR'],
    defaultLocale: 'en-GB',
  },
```
}

Full reference: [Page Router Internationalization](https://nextjs.org/docs/pages/building-your-application/routing/internationalization)
